pkg/syncer/metadatasyncer.go

```diff
@@ -346,7 +346,7 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 			isStorageQuotaM2Enabled := metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.StorageQuotaM2)
 			go func() {
 				if isStorageQuotaM2Enabled {
-					// TODO: Wait for fullSync to create CnsVolumeInfo CRs if necessary
```

Comment:
CnsVolumeInfo CRs are created at the time of CreateVolume, so there is no need to wait for fullSync to create it.